### PR TITLE
#3121: fix nav item alignment on small screen

### LINKED ISSUE #3121: Open Public Marketplace link alignment is off sometimes
Screenshot from a users computer:

image

### PR DESCRIPTION
**Pull request by abdullah4244**
Fixes #3121 
https://www.loom.com/share/0a8eec8857474b43b3dc2b6731e48fbf

### CODE REVIEW

**Comment by twschiller** [comment]
@mnholtz could you help take a look?

File: src/options/pages/blueprints/ListFilters.module.scss [hunk 1]
```diff
@@ -6,4 +6,7 @@
     font-size: 0.8rem;
     color: #bba8bff5;
   }
+  .nav {
+    flex-direction: column;
```
Comment (mnholtz) [inline src/options/pages/blueprints/ListFilters.module.scss:10]: nit: could we pull this out of `.root` to be at top-level? In general we try to keep our css modules un-nested where we can.

File: src/options/pages/blueprints/ListFilters.tsx [hunk 2]
```diff
@@ -129,7 +129,7 @@ function ListFilters({ teamFilters, tableInstance }: ListFiltersProps) {
           </Nav.Item>
         ))}
       </Nav>
-      <Nav>
+      <Nav className={styles.nav}>
```
Comment (mnholtz) [inline src/options/pages/blueprints/ListFilters.tsx:132]: nit: can we add this class to the other Nav component above, for consistency? [we are currently using the bootstrap "flex-column" class name](https://github.com/pixiebrix/pixiebrix-app/blob/d4451f2c986b7c1bd73bef91303c159d6385f0fb/pixiebrix-extension/src/options/pages/blueprints/ListFilters.tsx#L65-L66).

**Review by mnholtz — APPROVED** [review]
Thanks @abdullah4244 ! Looks good to me. Left some nit comments, but feel free to merge when ready.

**Comment by twschiller** [comment]
@abdullah4244 when this is ready, please go ahead and merge it in!

**Pull request merged**